### ISSUE TITLE
Auth not working

### ISSUE THREAD

**Issue by TiTi** [jeremyfa/node-redis-dump#10]
-a <auth> is not working:

```
E:\Redis\node_modules\redis-dump\node_modules\redis\index.js:506
                throw callback_err;
                ^

Error: Ready check failed: NOAUTH Authentication required.
    at RedisClient.on_info_cmd (E:\Redis\node_modules\redis-dump\node_modules\re
dis\index.js:319:35)
    at Command.callback (E:\Redis\node_modules\redis-dump\node_modules\redis\ind
ex.js:367:14)
    at RedisClient.return_error (E:\Redis\node_modules\redis-dump\node_modules\r
edis\index.js:502:25)
    at RedisReplyParser.<anonymous> (E:\Redis\node_modules\redis-dump\node_modul
es\redis\index.js:262:14)
    at emitOne (events.js:77:13)
    at RedisReplyParser.emit (events.js:169:7)
    at RedisReplyParser.send_error (E:\Redis\node_modules\redis-dump\node_module
s\redis\lib\parser\javascript.js:266:14)
    at RedisReplyParser.execute (E:\Redis\node_modules\redis-dump\node_modules\r
edis\lib\parser\javascript.js:125:22)
    at RedisClient.on_data (E:\Redis\node_modules\redis-dump\node_modules\redis\
index.js:478:27)
    at Socket.<anonymous> (E:\Redis\node_modules\redis-dump\node_modules\redis\i
ndex.js:79:14)

```

I have thoses versions:

┬ redis-dump@0.1.9
├── async@1.5.2
├─┬ optimist@0.3.7
│ └── wordwrap@0.0.3
└── redis@0.7.3

Fix in node_modules\redis-dump\bin\dump.js :

Replace this part :

```
RedisDumper = (function() {
    function RedisDumper(arg) {
      var auth, host, port;
      port = arg.port, host = arg.host, auth = arg.auth;
      if (auth != null) {
        this.db = redis.createClient(port, host, {
          auth_pass: auth
        });
      } else {
        this.db = redis.createClient(port, host);
      }
    }
```

with:

```
RedisDumper = (function() {
    function RedisDumper(arg) {
      var auth, host, port;
      port = arg.port, host = arg.host, auth = arg.auth;
      this.db = redis.createClient(port, host);
      if (auth != null) {
        this.db.auth(auth);
      }
    }
```

**Comment by keen99** [comment]
+1

```
%% redis-dump --help
redis-dump 0.1.9
```

```
%% redis-dump -h host -p port -a pass
/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:506
                throw callback_err;
                ^

Error: Ready check failed: NOAUTH Authentication required
    at RedisClient.on_info_cmd (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:319:35)
    at Command.callback (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:367:14)
    at RedisClient.return_error (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:502:25)
    at RedisReplyParser.<anonymous> (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:262:14)
    at emitOne (events.js:90:13)
    at RedisReplyParser.emit (events.js:182:7)
    at RedisReplyParser.send_error (/usr/local/lib/node_modules/redis-dump/node_modules/redis/lib/parser/javascript.js:266:14)
    at RedisReplyParser.execute (/usr/local/lib/node_modules/redis-dump/node_modules/redis/lib/parser/javascript.js:125:22)
    at RedisClient.on_data (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:478:27)
    at Socket.<anonymous> (/usr/local/lib/node_modules/redis-dump/node_modules/redis/index.js:79:14)
```

**Comment by afgarcia86** [comment]
+1

**Comment by jeremyfa** [comment]
It should work now. (released a new version `0.1.10` on NPM)
